### DISCUSSION TITLE
DSEP Schema Proposal: Agent Rating

### DISCUSSION THREAD

**Discussion by joffinjoy** [beckn/DSEP-Specification#78] [Ideas]
Current schema supports rating for: **Enumerated Entities, Providers, Items and fulfilment services**. But it is imperative that we should have agent level rating since it is a very widely used criteria for session selection. So far none of the previously mentioned rating allows for this. 

We should be able to search for sessions with filtering conditions like: "Taken by a mentor with rating higher than 4".

cc: @aks30, @vijiurs, @kiranharidas187, @core-wg-admin

**Comment by ravi-prakash-v** [comment]
@joffinjoy we discussed this on call. Can you post the recommendations that we arrived at here?

**Reply by aks30** [reply]
@core-wg-admin We discussed we can introduce the rating object to Fulfillment -> Agent -> Person

**Comment by joffinjoy** [comment]
@core-wg-admin I have raised a PR (https://github.com/beckn/protocol-specifications/pull/325) for adding **rating** to agent schema. 

In accordance with your suggestion we came to the conclusion that rating is better added as an agent attribute than that of the person. 

cc: @aks30, @vijiurs, @kiranharidas187
